internal/pkg/service/common/utctime/utctime.go

```diff
@@ -13,6 +15,10 @@ type (
 	UTCTime time.Time
 )
 
+func (v UTCTime) HandleDeepCopy(_ deepcopy.TranslateFn, _ deepcopy.Path, _ deepcopy.VisitedPtrMap) (UTCTime, deepcopy.CloneFn) {
+	return v, nil
+}
+
```

Comment:
Unrelated small change, `deepcopy` now can copy `UTCTime` time.

This method is required for all types with a private field.

Reply:
In this case, the value is simple, and we only copy it.